Favot/openai-translation-updater
# Create github template

## Description

This PR introduces a set of templates to streamline the process of creating Pull Requests, Issues, and Discussions within the repository. These templates will provide a structured format, making it easier for contributors to communicate effectively and maintain consistency across the repository.

### Changes

- Added a `PULL_REQUEST_TEMPLATE.md` to guide the creation of new Pull Requests.
- Introduced Issue templates:
  - Bug Report Template for reporting bugs.
  - Feature Request Template for proposing new features.
  - Custom Issue Template for other issues.
- Set up Discussion templates:
  - General Discussion Template to facilitate general discussions.
  - Q&A Template for asking questions and getting help from the community.

### Testing

- Navigate to the `.github` directory and review the template files to ensure they meet the project's requirements.
- Try creating a new Issue or Discussion on GitHub and verify that the new templates are available and functioning as expected.


---

By implementing these templates, we aim to foster clear, structured communication and make the repository more contributor-friendly. Please review the changes and provide feedback on any additional adjustments needed.
